projects/storefrontlib/cms-components/myaccount/my-account-v2/my-account-v2-navigation/my-account-v2-navigation.component.ts

```diff
@@ -12,4 +12,6 @@ import { NavigationComponent } from '../../../navigation';
   templateUrl: './my-account-v2-navigation.component.html',
   standalone: false,
 })
-export class MyAccountV2NavigationComponent extends NavigationComponent {}
+export class MyAccountV2NavigationComponent extends NavigationComponent {
+  isMyAccount = true;
+}
```

Comment:
```suggestion
export class MyAccountV2NavigationComponent extends NavigationComponent {}
```

Looks like this variable does not have a purpose, we should be able to remove it.